app.go

```diff
@@ -461,6 +473,20 @@ func (app *App) StopTimeout() time.Duration {
 	return app.stopTimeout
 }
 
+// DotGraph provides a graph of dependencies in the container as a string in DOT
+// syntax.
+func (app *App) DotGraph() DotGraph {
```

Comment:
Why export this? I would err on the side of not exposing new APIs if possible.

Reply:
Done.